bakery/src/cli/execute.js

```diff
@@ -325,6 +325,47 @@ const tasks = {
       }
     }
   },
+  link: (parentCommand) => {
```

Comment:
To be consistent with the other commands such as `bake-meta` etc., I think (assuming we leave this called `link-book`), we should change this to `'link-book'`